### PR TITLE
Release 4.0.4

### PR DESCRIPTION
**Pull request by neonmei**
For details, see corresponding Release issue #337

### CODE REVIEW

**Comment by neonmei** [comment]
Packages and other artifacts not yet released, so tests should fail.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Puppet v4.0.4
+
+### Added
+
+- Update to Wazuh version 4.0.4
```
Comment (xr09) [inline CHANGELOG.md:8]: Should we add a link to core changelog here as well?

Comment (Zenidd) [reply CHANGELOG.md]: Done on 6291ae4f54703523c145b2fbd2df52ab2de1f97f, thanks!

**Review by xr09 — APPROVED** [review]
LGTM!

**Review by Zenidd — APPROVED** [review]
LGTM!

**Pull request merged**